### ISSUE TITLE
[Discuss] Simplify code example usage, remove "View in Console"

### ISSUE THREAD

**Issue by cjcenizal** [elastic/docs#1238]
The impetus behind this proposal is multi-faceted:

* I can't seem to get "View in Console" to work for me (https://github.com/elastic/kibana/issues/9396 describes my experience)
* As a workaround to this feature, I can copy the example, navigate to Console and paste it in
* If this feature is this difficult to use and the workaround is so simple, then is the feature worth it?

I suggest that we can simplify the user experience and deliver the same value by removing this feature, adding "cURL" as one of the language options, and presenting a single "Copy" button that will copy the content of the example in whichever language it's presented in. Users can paste it wherever they want to use it

### Current
![image](https://user-images.githubusercontent.com/1238659/66000524-70101a00-e454-11e9-9ad9-af5fa7d20054.png)

### Proposed
![image](https://user-images.githubusercontent.com/1238659/66000616-a9e12080-e454-11e9-9c91-6ed0ec64dee1.png)

CC @nik9000 @debadair @111andre111

**Comment by cjcenizal** [comment]
If we have any usage statistics on the "Open in Console" feature that show that it's in heavy use then I think we should keep it.

**Comment by debadair** [comment]
Given that much of our docs traffic is driven from search links deep into the ES reference, “View in Console” makes a lot of assumptions that aren’t very user friendly. Everybody understands copy-paste. 

When there was just Console & cURL, the dual links made sense & were convenient. With more flavors of examples, one consistent copy-paste mechanism for what’s visible makes a lot of sense.

I agree that we should check the click traffic, but conceptually, this feels like the right thing to do.

**Comment by nik9000** [comment]
We get a couple of thousand hits to view in console a week. I isn't a ton, but it isn't nothing.

I'd love to have less stuff to support though.

**Comment by dedemorton** [comment]
One perhaps unmeasurable benefit of having the link is that it helps users make a connection between the console and the syntax they see for curl commands in the docs. Perhaps the benefit is too minor to be of concern...but figured it was worth pointing out. If the link is fragile, it probably does more harm than good, tho.

**Comment by 111andre111** [comment]
Just still to add here what I discuss with CJ:
Being a customer I have used this feature a lot.
But in theory if this feature produces too much overhead...
In my opinion this is just too bad documented.
Even less people I think know about the possibility that you can just copy the curl and then paste the whole string to kibana dev console and it get converted to the dev console request, that would be at least a workaround, which whould be documented better too.

And that the view in console is configurable for changing the host and port to another matching configuration is unknown for lots of people too.

**Comment by cjcenizal** [comment]
CC @jloleysens since this could affect Console.

**Comment by debadair** [comment]
I agree with the points @111andre111 makes above--if we decide to keep View in Console, we need to do a better job of highlighting its usefulness and configurability. And even if we don't, we need to make a clearer connection between "Console" and the Dev Console in Kibana. 

When it was View in Sense, that was at least a unique name for a standalone Elastic-related tool. Console is a very generic, much-overloaded term--I suspect a lot of folks have no idea what console we're talking about. As @dedemorton pointed out, highlighting that connection in some fashion is desirable.

Would it be possible to have a different tooltip for a generic "Copy" link that provided a bit more info depending on the selected language, for example "Copy example for Kibana Dev Console"?

**Comment by nik9000** [comment]
> Would it be possible to have a different tooltip for a generic "Copy" link that provided a bit more info depending on the selected language, for example "Copy example for Kibana Dev Console"?

If we had a generic copy link we could certainly do that.
